core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -28,8 +29,7 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
           <span translate="I18N_LIBRARY_NO_OBJECTIVE">
           </span>
         </div>
-
-        <ul layout="row" class="metrics" layout-align="space-between center">
+        <ul ng-if="isCollectionPreviewTile === undefined || !isCollectionPreviewTile" layout="row" class="metrics" layout-align="space-between center">
```

Comment:
Oh, I completely missed this, thanks @Oishikatta. Yeah -- I'm also wondering, why can't it just be `ng-if="!isCollectionPreviewTile"`?